eng/pipelines/templates/stages/archetype-cpp-release.yml

```diff
@@ -21,8 +21,8 @@ stages:
             environment: github
 
             pool:
-              name: azsdk-pool-mms-win-2019-general
-              vmImage: MMS2019
+              name: azsdk-pool-mms-win-2022-general
+              vmImage: MMS2022
```

Comment:
We don't use `vmImage` in the places where this is changed. Let's remove it in these instances where it's updated to `MMS2022`

Reply:
I'll clean this up in another issue. https://github.com/Azure/azure-sdk-for-cpp/issues/4211